packages/jana2/package.py

```diff
@@ -58,14 +58,18 @@ class Jana2(CMakePackage):
 
     variant("root", default=False, description="Use ROOT for janarate.")
     variant("zmq", default=False, description="Use zeroMQ for janacontrol.")
+    variant("podio", default=False, description="Use Podio.")
```

Comment:
Podio support only introduced in 2.0.9.
```suggestion
    variant("podio", default=False, description="Use Podio.", when="@2.0.9:")
```